locals.tf

```
@@ -80,14 +80,15 @@ locals {
         adminPassword = "${replace(local.grafana.admin_password, "\"", "\\\"")}"
         "grafana.ini" = {
           "auth.generic_oauth" = merge({
-            enabled       = true
-            allow_sign_up = true
-            client_id     = "${replace(local.grafana.oidc.client_id, "\"", "\\\"")}"
-            client_secret = "${replace(local.grafana.oidc.client_secret, "\"", "\\\"")}"
-            scopes        = "openid profile email"
-            auth_url      = "${replace(local.grafana.oidc.oauth_url, "\"", "\\\"")}"
-            token_url     = "${replace(local.grafana.oidc.token_url, "\"", "\\\"")}"
-            api_url       = "${replace(local.grafana.oidc.api_url, "\"", "\\\"")}"
+            enabled                  = true
+            allow_sign_up            = true
+            client_id                = "${replace(local.grafana.oidc.client_id, "\"", "\\\"")}"
+            client_secret            = "${replace(local.grafana.oidc.client_secret, "\"", "\\\"")}"
+            scopes                   = "openid profile email"
+            auth_url                 = "${replace(local.grafana.oidc.oauth_url, "\"", "\\\"")}"
+            token_url                = "${replace(local.grafana.oidc.token_url, "\"", "\\\"")}"
+            api_url                  = "${replace(local.grafana.oidc.api_url, "\"", "\\\"")}"
+            tls_skip_verify_insecure = var.cluster_issuer == "ca-issuer" ? true : false
```

Actually I guess read this too fast the first time. You don't need a ternary operator if you're just returning the same value as the condition...
```suggestion
            tls_skip_verify_insecure = var.cluster_issuer == "ca-issuer"
```